server/sonar-server/src/test/java/org/sonar/server/user/UserUpdaterTest.java

```diff
@@ -79,6 +78,9 @@
   @Mock
   NewUserNotifier newUserNotifier;
 
+  @Mock
```

Comment:
Not here, see the ``NewUserNotifier`` stuff above - the object graph below is just too heavy. Besides, we don't want to couple this WS with the implementation of ``SecurityRealmFactory``.